src/_albedo/audio/devices/Devices.models.ts

```diff
@@ -4,16 +4,11 @@
 // I predict future differences between input and output devices.
 // Input devices will probably have info about supported formats or something.
 
-
 export interface DeviceBase {
   id: string;
   name: string;
 }
 
-export interface InputDevice extends DeviceBase {
```

Comment:
If you want to leave placeholders, VSCode has an option to "opt out" lint errors for specific lines. It's okay to use that as long we add a TODO, so we don't forget any :)

Reply:
That all sounds good to me. I'm trying not to opt out of any. So far, I've only suppressed one, which had do to with naming. Because dB is at the start of a class name.